MoveNet_Vs_PoseNet.ipynb

```diff
@@ -272,7 +274,7 @@
     "        pose_kps[i,0] = int(remap_pos[0] + offset_data[max_val_pos[0],max_val_pos[1],i])\n",
     "        pose_kps[i,1] = int(remap_pos[1] + offset_data[max_val_pos[0],max_val_pos[1],i+joint_num])\n",
     "        max_prob = np.max(joint_heatmap)\n",
-    "        pose_kps[i,3] = max_prob\n",
+    "        pose_kps[i,3] = sigmoid(max_prob)\n",
```

Comment:
@bhavikapanara is this the fix of the formula that outputs 50% score for posenet when it should be 0?

Reply:
@bhavikapanara  something does not look right in the posetnet score calculation formula. Even after the sigmoid function the results appear wrong. In most cases the scores are either exactly 0 (which is almost impossible) or above 0.9 (which is unlikely) when the corresponding movenet scores are more evenly distributed as expected. Please review closer the posenet score calculation formulas and test them thoroughly. Even better use the formulas from the already tested code we use for posenet.

As a related point, this notebook should not be rewriting the model output calculations. Instead the notebook should simply reference high level functions from the standalone fall-detection library code. Each AI model needs to have its own submodule that developers can use to swap out alternative implementations (posenet, movenet, mediapipe, etc.) with no impact to code using the fall-detection library API.

Otherwise the notebook becomes a QA tool for model functions instead of a simple performance comparison tool.

As we add more models in the future, it should be simple enough to plug them into this notebook for side by side comparison.